.github/workflows/ci.yml

```diff
@@ -42,62 +42,20 @@ jobs:
   spell-check:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Parent Repo
-        uses: actions/checkout@v2
-        with:
-          ref: main
-          repository: aws/aws-iot-device-sdk-embedded-C
-          path: main
       - name: Clone This Repo
         uses: actions/checkout@v2
+      - name: Run spellings check
+        uses: FreeRTOS/CI-CD-Github-Actions/spellings@main
         with:
-          path: ./tcp
-      - name: Install spell
-        run: |
-          sudo apt-get install spell
-          sudo apt-get install util-linux
-      - name: Check spelling
-        run: |
-          PATH=$PATH:main/tools/spell
-          # Make sure that the portable directory is not included in the spellcheck.
-          sed -i 's/find $DIRNAME/find $DIRNAME -not -path '*portable*'/g' main/tools/spell/find-unknown-comment-words
```

Comment:
How is the spellcheck now ignoring the 'portable' directory with this change?

Reply:
The spell check passes while not ignoring this directory - I am also confused as to why this directory was excluded to begin with.

Reply:
@AniruddhaKanhere  , @Skptak  I am not sure what was reasoning about skipping spell check for portable layer? Do you have any idea about it? With removing the portable layer skip i can see spell checker is failing with a lot of errors from portable layer